go/pkg/services/logcollector/test/mock/mock.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 Iguazio
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mock
```

Comment:
that's a "nop" not a "mock".

On `Mock` client you assert / validate calls
on `Nop` client you insert dummy logic for test purposes, or avoiding complex / business logics